Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM rocker/r-ver:4.0.2
+
+USER root
```

Comment:
TODO: See if this is necessary

Reply:
See https://github.com/2DegreesInvesting/pactaCore/pull/21